code/modules/clothing/spacesuits/syndi.dm

```diff
@@ -1,11 +1,10 @@
 //Regular syndicate space suit
 /obj/item/clothing/head/helmet/space/syndicate
 	name = "red space helmet"
-	desc = "Top secret Spess Helmet."

```

Comment:
Why delete it’s description?

Reply:
was doubled, it had two desc and that one was overwritten